### ISSUE TITLE
Events/Alarms Framework

### ISSUE THREAD

**Issue by jeff-yin** [sonic-net/SONiC#1549]

**Comment by zhangyanzhao** [comment]
duplicated with 1409 item, close this one.

**Comment by zhangyanzhao** [comment]
@venkatmahalingam  @jeff-yin  do we still need this issue for 202511 release? I see the related code PRs have been merged to 202505 release.

**Comment by jeffyin-dell** [comment]
@zhangyanzhao yes! The original code PRs you're referring to were backed out. I guess there was a build failure, although I'm not sure whether it was actually related. Anyway, please check out https://github.com/sonic-net/sonic-buildimage/pull/22617
